hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/SerializationUtilTest.java

```diff
@@ -102,6 +108,21 @@ public void testReadWriteNullableBoolean_whenInvalid()
         deserialize(bytes);
     }
 
+    @Test
+    public void testGetInterfacesCalledVerified() {
+
```

Comment:
minor: unnecessary newline

Reply:
Thank you for the suggestion, I have updated my pull request with a new commit to fix this.